new-website/deepchem/pages/_document.js

```diff
@@ -37,7 +37,7 @@ const Document = () => {
           referrerPolicy="no-referrer"
         />
       </Head>
-      <body>
+      <body className="flex flex-col min-h-screen">
```

Comment:
Can you explain what these changes do?

Reply:
This sets the necessary properties of the body to flex, flex column and full view height to get the footer to stick to bottom of the screen as explained above.